### PR TITLE
fix: Mode was not public

### LINKED ISSUE #57: Mode is no exposed publicly
Hey @erwanvivien I was trying to use the new mode function, but looks like Mode is not exposed publicly because `encode` is private module.

I can do the PR but would you prefer to alias mode, in the `qr` module or expose `encode` as a public module?

### PR DESCRIPTION
**Pull request by erwanvivien**
Fixes #57

### CODE REVIEW

**Review by praveenperera — APPROVED** [review]

File: src/encode.rs [hunk 1]
```diff
@@ -172,7 +180,7 @@ pub(crate) fn ascii_to_alphanumeric(c: u8) -> usize {
         b'.' => 42,
         b'/' => 43,
         b':' => 44,
-        _ => panic!("Character '{}' should not occur", c as char), // unreachable!()
+        _ => panic!("Unexpected haracter '{}' in Alphanumeric mode", c as char),
```
Comment (praveenperera) [inline src/encode.rs:183]:
```suggestion
        _ => panic!("Unexpected character '{}' in Alphanumeric mode", c as char),
```

**Comment by erwanvivien** [comment]
fast_qr v0.12.5 is deployed to crates.io

**Comment by praveenperera** [comment]
@erwanvivien thanks! it works, I updated bbqr with it

**Comment by erwanvivien** [comment]
🙏

**Pull request merged**